cmd/umserver/main.go

```diff
@@ -4,6 +4,83 @@
 // All users data will be stored in a database.
 package main
 
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+const GracefulShutdownTimeOut = 10 * time.Second
+
 func main() {
```

Comment:
why you made this in main?

Reply:
I agree with you. I will move it into separate file, called gracefulshutdown.go

Thanks!

Reply:
moved back to main.go as go build main.go stop worked

Reply:
Can't you just use just command "go build ./"  in cmd/umserver directory instead?)

Reply:
If you try to build only main go ignoring  gracefulshutdown.go it must fail

Reply:
You are right, it will be work also. )

Reply:
@i3odja, your IDE is not something that should impact code structure, configure it properly to avoid such problems.

Reply:
can I leave implementation of graceful shutdown in main.go?

Reply:
Well yes but actually no. You can leave it in main.go but no one will approve this

Reply:
Finally, I have done it. I moved shutdown into another file